### ISSUE TITLE
Minimal example for testing pretrained model

### ISSUE THREAD

**Issue by marco-rudolph** [BrandonHanx/FAME-ViL#11]
Hey,

great work! :)

Could you provide a minimal example for loading the given pretrained fashionclip model and get the embedding for an example image?

Thanks in advance, 
Marco

**Comment by yunusskeete** [comment]
This would be really helpful!

**Comment by Wayfarer123** [comment]
> Hey,
> 
> great work! :)
> 
> Could you provide a minimal example for loading the given pretrained fashionclip model and get the embedding for an example image?
> 
> Thanks in advance, Marco

Did you manage to make it work? Even the script suggested in the readme doesn't work for me for some reason

**Comment by marco-rudolph** [comment]
No, I gave up at some point and hoped for a response here. And I did not try to test the example script.
